### PR TITLE
params orientation & page-size

### PR DESCRIPTION
**Pull request by dvdn**
To take new params :
- orientation
- page size

### CODE REVIEW

**Comment by ooga** [comment]
Readme :)

File: web/index.php [hunk 1]
```diff
@@ -20,6 +20,17 @@
...
     check_get_params($snappy);
+
+    // Page size (international ISO standard : Letter, A0, A1, A2, ...)
+    if (isset($_GET['page-size'])) {
+        $snappy->setOption('page-size', $_GET['page-size']);
+    }
+
+    // Page orientation (portrait,landscape)
+    if (isset($_GET['orientation'])) {
+        $snappy->setOption('orientation', $_GET['orientation']);
```
Comment (vinzentt) [inline web/index.php]: no need to set those params explicitly, as the Readme says: "all wkhtmltopdf options are available" thanks to this function https://github.com/CanalTP/pdfGenerator/blob/master/web/index.php#L77

**Comment by vinzentt** [comment]
Normally page-size and orientation should work without this PR

**Comment by dvdn** [comment]
Useless

**Pull request closed without merging**